### PR TITLE
chore(liveness): add more detail about Amplify CLI options to native docs

### PR DESCRIPTION
**Pull request by tjleing**
#### Description of changes

Added callout that Cognito does not mean using it as auth provider, just for Rekognition access.  Added detail on CLI import, simpler `amplifyconfiguration.json`, moved credentials provider section to where it's more applicable

#### Checklist

- [x] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [x] PR description included
- [x] Relevant documentation is changed or added (and PR referenced)
- [x] `yarn test` passes and tests are updated/added
- [x] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import { Tabs, TabItem } from '@aws-amplify/ui-react';
+import { Fragment } from '@/components/Fragment';
 
 There are three methods to setup the Amazon Cognito resources needed for Amplify Auth. The most common is the Amplify CLI create flow which asks a series of questions and will configure both a User Pool and Identity Pool automatically. The second option is the Amplify CLI import flow which adds an existing Cognito resource into Amplify. Finally the third is to reuse or create a Cognito Identity Pool manually and to add it into your application.
 
+<Fragment>
```
Comment (phantumcode) [inline docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx:6]: Can you add the same cognito-callout fragment to `quick-start-pull.swift.mdx`?

File: docs/src/pages/[platform]/connected-components/liveness/credentials-provider.android.mdx [hunk 2]
```diff
@@ -1,28 +0,0 @@
-## Auth
```
Comment (thaddmt) [inline docs/src/pages/[platform]/connected-components/liveness/credentials-provider.android.mdx:1]: Can we delete this file if it is unused?

**Review by thaddmt — COMMENTED** [review]
LGTM other than small comments!

File: docs/src/pages/[platform]/connected-components/liveness/credentials-provider.android.mdx (hunk 2 above)
Comment (tjleing) [reply docs/src/pages/[platform]/connected-components/liveness/credentials-provider.android.mdx:1]: Breaks imports I think, that's why the React one is empty as well.  When Swift deletes that file as well we can remove the files and the import

File: docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.swift.mdx [hunk 3]
```diff
@@ -1,7 +1,98 @@
...
+  </TabItem>
+  <TabItem title="Self Managed Cognito Resource">
+> Use this option if you already have a Cognito identity/user pools that you do not want to import to Amplify, or want to manage Cognito resources yourself or with a 3rd party resource management tool.
+
+If you already have Cognito set up or do not want to use the Amplify CLI to generate Cognito resources, you can follow the documentation in the [existing resources tab](https://docs.amplify.aws/lib/auth/getting-started/q/platform/android/#set-up-backend-resources).
+
+_If you are manually setting up an identity pool in the Cognito console you can follow [this guide](https://docs.aws.amazon.com/cognito/latest/developerguide/getting-started-with-identity-pools.html#create-identity-pool). When setting up the identity pool ensure that access to unauthenticated identities is enabled._
+
+When initially configuring Amplify (assuming you are using no pieces of Amplify other than the FaceLivenessDetector) you can manually create an `amplifyconfiguration.json`and add it your project’s main directory in this form:
```
Comment (tjleing) [inline docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.swift.mdx:32]:
```suggestion
When initially configuring Amplify (assuming you are using no pieces of Amplify other than the FaceLivenessDetector) you can manually create an `amplifyconfiguration.json` and add it your project’s main directory in this form:
```

**Review by thaddmt — APPROVED** [review]
LGTM

**Review by reesscot — APPROVED** [review]

**Review by thaddmt — APPROVED** [review]

**Review by thaddmt — DISMISSED** [review]
LGTM in code, waiting to review on the docs

**Comment by reesscot** [comment]
https://native-liveness-cli-alternative-docs.d22cn4asamk913.amplifyapp.com/

**Review by thaddmt — APPROVED** [review]

**Review by wlee221 — APPROVED** [review]
🛥️

**Pull request merged**